spec/models/action_spec.rb

```diff
@@ -2,11 +2,81 @@
 
 describe Action do
   let(:page) { create :page }
+  let(:member) { create :member }
 
   describe 'on create' do
-    subject{ create(:action, page_id: page.id) }
+    describe 'member donor_status' do
+
+      describe 'when member is nondonor' do
+        before :each do
+          expect(member.donor_status).to eq 'nondonor'
```

Comment:
To me, it clarifies things to have before blocks that establish the expectations for all the tests in that block, but I'm willing to conform to a different style if you cite a source.